### ISSUE TITLE
Add ability to toggle between changed modules & affected dependencies

### ISSUE THREAD

**Issue by GabrielBottari** [ismaeldivita/change-tracker-plugin#14]
It would be nice to be able to specify a command-line flag to toggle between **changed modules + affected dependencies** and **only changed modules**. This comes useful for tasks like linting, which won't affect top-level modules in case of changes.

[The AffectedModuleDetector plugin has an example here on a similar feature](https://github.com/dropbox/AffectedModuleDetector#running-all-changed-projects), but I believe your plugin is still more powerful, which is why it would be nice to get such a feature here!

**Comment by ismaeldivita** [comment]
Yeah, that would be good. I'm already working on a version that will support this and also improve how we trigger tasks for different module types, bear with me.
